docs/python/generators/examples_helpers.py

```
@@ -86,6 +86,105 @@ def _find_unquoted_word_on_nontrivial_line(word: str, text: str) -> bool:
+            if ln in ('{', '}', '#endif', '#if 0') or ln.startswith('#if') or ln.startswith('#define'):
+                continue
+            return (m, line_idx)
+    # fallback to strong patterns
+    for p in (strong_patterns or []):
+        for m in re.finditer(p, text):
+            if _is_in_double_quotes(text, m.start()):
+                continue
+            pos = m.start()
+            cum = 0
+            line_idx = 0
+            for i, l in enumerate(lines):
```

Ruff (0.15.21)

[error] Ambiguous variable name: `l`

(E741)